templates/router/header.conf.erb

```diff
@@ -22,7 +22,7 @@
 
 router {
     id: <%= scope['qpid::router::router_id'] %>
-    mode: <%= scope['qpid::router::router_mode'] %>
+    mode: <%= scope['qpid::router::mode'] %>
```

Comment:
This looks like an unrelated bug fix. For the changelog it may be good to have it separate. In case we want to cherry pick it.

Reply:
I can extract it. This change unearthed the bug since `router_mode` got removed from params.pp this removed the parameter and tests pointed out the mismatch.

Reply:
https://github.com/theforeman/puppet-qpid/pull/153